docker-compose.ha.yml

```diff
@@ -197,65 +201,35 @@ services:
     - FEATURE_PERMISSION_PROFILE_LABELS=false
     - FEATURE_BPM=true
     - FEATURE_IMPORT_WITH_SAME_AND_UPPERRANK=true
+    - FEATURE_REPOFOLDERLIST=[\"Datafiles\"]
     logging:
       <<: *logging
     networks:
       default:
         aliases:
           - "msa_ui"
 
-  msa-event:
+  msa-rsyslog:
     depends_on:
-      - db
-      - msa-es
-      - msa-dev
-      - msa-sms
+      - msa-event
+    image: ubiqube/msa2-rsyslog:ce35fbb9c7441f1c4041af6e431a0cfae2809785
     healthcheck:
       timeout: 5s
-      retries: 10
+      retries: 2
       interval: 10s
       start_period: 30s
-      test: ["CMD-SHELL", "/etc/init.d/ubi-sms status | grep -q 'service seems UP' || exit 1"]
-    image: openmsa/openmsa:msa2-event-2.8.10-51bd24c1e5a4fb438fa21df5b4595ae1a5d003a2
-    environment:
-      <<: *es-configuration
-      CONTAINER_DOCKNAME: "{{.Task.Name}}.{{.Node.Hostname}}"
-      HOST_HOSTNAME: "{{.Node.Hostname}}"
-    logging:
-      driver: "json-file"
-      options:
-        mode: non-blocking
-        max-buffer-size: "4m"
-        max-size: "20m"
-        max-file: "5"
-    volumes:
-      - "/mnt/NASVolume/msa_dev:/opt/devops/"
-      - "/mnt/NASVolume/msa_bulkfiles:/opt/sms/spool/parser"
-      - "/mnt/NASVolume/msa_bulkfiles_err:/opt/sms/spool/parser-error"
+      test: ["CMD-SHELL", "top -p 1 -bn1 | grep rsyslog | awk '{if ($$9 > 99) exit 1; else exit 0;}'"]
```

Comment:
Is it working ?

Reply:
@UBiqube-ydu 
can you test and select the command to use ?